plugins/com.google.cloud.tools.eclipse.appengine.deploy/src/com/google/cloud/tools/eclipse/appengine/deploy/standard/StandardDeployPreferencesConverter.java

```diff
@@ -0,0 +1,35 @@
+package com.google.cloud.tools.eclipse.appengine.deploy.standard;
+
+import com.google.cloud.tools.appengine.api.deploy.DefaultDeployConfiguration;
+
+public class StandardDeployPreferencesConverter {
+
+  private StandardDeployPreferences preferences;
+
+  public StandardDeployPreferencesConverter(StandardDeployPreferences preferences) {
+    this.preferences = preferences;
+  }
+
+  public DefaultDeployConfiguration toDeployConfiguration() {
```

Comment:
I don't know. I don't feel it as a significant improvement, but I agree that this class and method could be better named.